frontend/ui/src/pages/Inbox/MessageInput/index.tsx

```diff
@@ -65,18 +65,24 @@ const MessageInput = (props: MessageInputProps & ConnectedProps<typeof connector
   const [selectedTemplate, setSelectedTemplate] = useState<SelectedTemplate | null>(null);
   const [disconnectedChannelToolTip, setDisconnectedChannelToolTip] = useState(false);
   const [selectedSuggestedReply, setSelectedSuggestedReply] = useState<SelectedSuggestedReply | null>(null);
+  const [closeIconWidth, setCloseIconWidth] = useState('');
+  const [closeIconHeight, setCloseIconHeight] = useState('');
 
   const textAreaRef = useRef(null);
   const sendButtonRef = useRef(null);
   const emojiDiv = useRef<HTMLDivElement>(null);
+  const templateSelectorDiv = useRef<HTMLDivElement>(null);
+  const removeTemplateButton = useRef(null);
 
   const handleChange = (e: React.ChangeEvent<HTMLTextAreaElement>): void => {
     setInput(e.target.value);
   };
 
   useEffect(() => {
     setInput('');
-    textAreaRef && textAreaRef.current.focus();
+    removeTemplateFromInput();
+
+    textAreaRef?.current?.focus();
   }, [conversation.id]);
 
```

Comment:
here fixed edge case when the user selects a template and then changes conversation (without sending or removing template before changing). 

--> When a user changes conversation, the template selector and input are reset.